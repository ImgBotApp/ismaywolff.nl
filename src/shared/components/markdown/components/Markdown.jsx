import styled from 'styled-components'
import { colors } from '../../../styles'

// Sets styles for rendered markdown, since components can't be used there
const Markdown = styled.div`
  & a {
    border-bottom: 1px solid ${colors.blue};
    color: ${colors.blue};
    text-decoration: none;
  }
`

export default Markdown
