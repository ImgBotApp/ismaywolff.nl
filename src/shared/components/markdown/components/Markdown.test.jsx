/**
 * @jest-environment node
 */

import 'jest-styled-components'
import React from 'react'
import { shallow } from 'enzyme'
import Markdown from './Markdown'

describe('<Markdown />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Markdown />)
    expect(wrapper).toMatchStyledComponentsSnapshot()
  })
})
