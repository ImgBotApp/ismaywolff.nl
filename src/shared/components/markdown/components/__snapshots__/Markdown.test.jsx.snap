// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Markdown /> renders correctly 1`] = `
.kGkrne a {
  border-bottom: 1px solid rgb(64,134,203);
  color: rgb(64,134,203);
  -webkit-text-decoration: none;
  text-decoration: none;
}

<div
  className="sc-bdVaJa kGkrne"
/>
`;
